.github/workflows/ci.yml

```diff
@@ -5,11 +5,20 @@ on:
     paths-ignore:
       - '**.md'
       - '**.rst'
+  pull_request:
+    paths-ignore:
+      - "**.md"
+      - "**.rst"
+
+concurrency:
+  group: ci-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
 
 jobs:
   tests:
     name: Tests
     runs-on: macos-latest
+    if: github.event_name != 'push' || github.event.ref == 'refs/heads/master'	
```

Comment:
I understand the desire to do this, but I have caught bugs that only happen on one OS, so I wan't it to still prove the binaries can be built as part of CI.

Reply:
ok

Reply:
Fixed in 162fd1e .